bin/clean_topics_vs_practices.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+import json
+
+
+def main():
+    with open("config.json", encoding="utf-8") as f:
+        config = json.load(f)
+
+    concepts = {c['slug'] for c in config['concepts']}
+
+    for practice_exercise in config['exercises']['practice']:
+        if practice_exercise['topics'] is None:
+            continue
+
+        practice_exercise['practices'].extend((topic for topic in practice_exercise['topics'] if topic in concepts))
+        practice_exercise['topics'] = [topic for topic in practice_exercise['topics'] if topic not in concepts]
+
+    with open("config.json", 'w', encoding="utf-8") as f:
+        json.dump(config, f, indent=2, ensure_ascii=False)
+        f.write('\n')
+
+    print("Updated config.json")
+
+
+main()
```

Comment:
```suggestion
if __name__ == '__main__':
    main()
```

This means it'll only run the `main` function when run as a script; it won't run it on import.

Reply:
Oh right, forgot that!